zap/src/main/dist/lang/Messages_es_ES.properties

```diff
@@ -123,6 +123,10 @@ api.home.links.header=<h2>Enlaces</h2>
 api.home.links.api.enabled=<li><a href\="/UI"> API local</a></li>
 api.home.links.api.disabled=<li>API local desactivada - Puede ser activada a trav\u00e9s de Herramientas / Opciones... /</li>
 api.home.links.online=<li><a href\="https\://www.owasp.org/index.php/ZAP">Pagina de ZAP</a></li>\n<li><a href\="https\://github.com/zaproxy/zaproxy/wiki/Introduction">ZAP Wiki</a></li>\n<li><a href\="https\://groups.google.com/group/zaproxy-users">Grupo de Usuarios de ZAP</a></li>\n<li><a href\="https\://groups.google.com/group/zaproxy-develop">Grupo de desarrolladores ZAP</a></li>\n<li><a href\="https\://github.com/zaproxy/zaproxy/issues">Reportar una falla</a></li>
+api.home.cacert=<h2>Evitar Advertencias de HTTPS</h2>\
```

Comment:
This should be reverted, only the main file should be changed. Translations happen in Crowdin.